class.jetpack-options.php

```diff
@@ -246,9 +304,17 @@ public static function delete_option( $names ) {
 		}
 
 		foreach ( array_intersect( $names, self::get_option_names( 'non_compact' ) ) as $name ) {
-			if ( ! delete_option( "jetpack_$name" ) ) {
-				$result = false;
+			if ( self::is_network_option( $name ) ) {
+				if ( ! delete_site_option( "jetpack_$name" ) ) {
```

Comment:
Could these lines be simplified as `$result = delete_site_option( "jetpack_$name" );` ?

[`delete_site_option()`](https://github.com/WordPress/WordPress/blob/9d7ea04936125041b224b227f48ad4194066b314/wp-includes/option.php#L1057) always returns a bool anyways.